### PR TITLE
Remove some deprecated parameters and add a few extra

### PR DESCRIPTION
**Pull request by mayorova**
Also this makes it closer to the [productized template](https://raw.githubusercontent.com/3scale/3scale-amp-openshift-templates/2.2.0.GA/apicast-gateway/apicast.yml).

### CODE REVIEW

File: openshift/apicast-template.yml [hunk 1]
```diff
@@ -122,6 +120,10 @@ parameters:
   value: apicast
   name: APICAST_NAME
   required: true
+- description: "Number of worker processes to start."
+  value: "1"
```
Comment (mikz) [inline openshift/apicast-template.yml]: This would actually cripple the deployment to 1 worker instead of automatically detecting available CPUs.

**Comment by mayorova** [comment]
So we need to do the same [here](https://github.com/3scale/3scale-amp-openshift-templates/blob/master/apicast-gateway/apicast.yml#L132) @mikz ?

**Comment by mikz** [comment]
@mayorova yep, that would be good. Otherwise the auto detection (https://github.com/3scale/apicast/pull/600) won't work.

**Comment by mayorova** [comment]
@mikz I opened an issue here to not forget about it: https://issues.jboss.org/browse/THREESCALE-1073

**Comment by mikz** [comment]
Thank you @mayorova! 👍

**Pull request merged**